.travis.yml

```diff
@@ -8,7 +8,7 @@ cache:
   - node_modules
 install:
 - sudo apt-get update && sudo apt-get install -y libcairo2-dev libpango1.0-dev libssl-dev libjpeg62-dev libgif-dev pkg-config
-- npm install canvas
```

Comment:
How come this isn't a `devDependency`? (Sorry if I already asked this)

Reply:
Canvas is weirdly difficult to install. Also it only seems to be missing on Travis; these tests seem to work for everyone locally. Running npm install canvas doesn't work (at least for me); it gives this error:
npm ERR! Linux 4.4.0-43-Microsoft
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "--save-dev" "canvas@1.6.7"
npm ERR! node v6.11.2
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE

npm ERR! canvas@1.6.7 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the canvas@1.6.7 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the canvas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs canvas
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls canvas
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /mnt/c/Users/DD/Git/scratch-paint-1/npm-debug.log

Reply:
I tested it and adding to dev deps is enough to fix Travis! It just makes npm install fail for me locally
https://github.com/LLK/scratch-paint/pull/189

Reply:
@fsih @rschamp i'd rather fix travis and fix the lack of dev dependency in separate pieces (it is more important to fix travis right now, and i'd rather doing so not break running npm install). to that end, the change to the travis yml looks good to me. 

However, @fsih it looks like there is a random update to the eslint version number? that probably shouldn't be in this PR.

Reply:
Oh just saw 189. Ignore me, whatever works to get travis running again is 👌